queries/roads.jinja2

```diff
@@ -3,6 +3,7 @@ SELECT
     gid AS __id__,
     {% filter geometry %}the_geom{% endfilter %} AS __geometry__,
     'naturalearthdata.com' AS source,
+    TODO_FIX_ME as min_zoom,
```

Comment:
We seem to include roads where `scalerank <= zoom`, so I think that effectively means `scalerank = min_zoom`.